bridge/client/app/_components/ktb-stage-badge/ktb-stage-badge.component.scss

```diff
@@ -6,7 +6,7 @@
 
   .dt-tag {
     background-color: $gray-200;
-    padding-right: 45px;
+    padding-right: 54px;
```

Comment:
Because the decimals are removed again in the bubbles, could you also revert the increased padding here?